test/plugins/scenarios/pulsar/config/excepted.yml

```diff
@@ -0,0 +1,142 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+  - serviceName: pulsar
+    segmentSize: 4
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: GET:/health
```

Comment:
Please delete the `GET:/health` checker.

Reply:
Done.

Reply:
I noticed that I have also retained this spans in other plugin's file, I'll delete it from future PR.